.env

```diff
@@ -22,3 +22,9 @@ LDAP_SEARCH_BASE="ou=people,dc=planetexpress,dc=com"
 LDAP_SEARCH_FILTER="(mail={{username}})"
 LDAP_SEARCH_ATTRIBUTES="mail, uid"
 LDAP_EMAIL_FIELDNAME=mail
+
+# API HOSTS
+CONFIG_HOST=artemis_configuration_1
```

Comment:
you do not need this in this form, you only contact "configuration". You should not care which container will serve you, so you can set: "CONFIG_HOST=configuration" simply